tests/regression_tests/triso/results_true.dat

```diff
@@ -1,2 +1,2 @@
 k-combined:
-1.701412E+00 3.180881E-02
+1.701412E+00 3.180881E-0
```

Comment:
Was this change intentional?

Reply:
Definitely a mistake -- I had updated the numbers following the changes made in PR, but then decided not to as the new numbers would have been generated using the new PRNG, so would not correspond to the current PRNG anyhow. Meant to roll back my change but must have clipped the 2 at the end. Great catch!!!